### PR TITLE
url for CP blog post

### PR DESCRIPTION
**Pull request by jhung0**
Should it be a clickable link? And just be written in 1 place?

### CODE REVIEW

**Comment by mcquin** [comment]
A clickable link would be great!

**Comment by jhung0** [comment]
I don't think it's possible with wx.MessageBox

**Comment by 0x00b1** [comment]
I found this example:

hunter

**Comment by 0x00b1** [comment]
I believe this uses `HyperlinkCtrl`.

File: cellprofiler/gui/imagesetctrl.py [hunk 1]
```diff
@@ -435,8 +435,8 @@ def on_update(self):
             wx.MessageBox(
```
Comment (mcquin) [inline cellprofiler/gui/imagesetctrl.py]: https://wxpython.org/docs/api/wx.HyperlinkCtrl-class.html

File: cellprofiler/gui/pipelinecontroller.py [hunk 2]
```diff
@@ -1166,7 +1166,7 @@ def __on_plateviewer(self, event):
         if len(image_numbers) == 0:
             self.display_plate_viewer_help(
```
Comment (mcquin) [inline cellprofiler/gui/pipelinecontroller.py]: You may also want to change the control defined in `display_plate_viewer_help` to support links.

**Comment by 0x00b1** [comment]
@jhung0 bump

**Comment by jhung0** [comment]
I know, I think I just need more time to understand wx

**Comment by 0x00b1** [comment]
@jhung0 bump. It’d be nice if @mcquin or I could merge this soon. @mcquin left you some useful notes and please reach out to either @mcquin or I if you need some additional help or guidance.

**Comment by mcquin** [comment]
Closing: inactivity

**Pull request closed without merging**